helpdesk-service/helpdesk-service-server/src/test/java/org/symphonyoss/symphony/bots/helpdesk/service/steps/MakerCheckerSteps.java

```diff
@@ -0,0 +1,243 @@
+package org.symphonyoss.symphony.bots.helpdesk.service.steps;
+
+import static junit.framework.TestCase.assertEquals;
+import static junit.framework.TestCase.assertNotNull;
+import static junit.framework.TestCase.assertNull;
+import static org.springframework.http.MediaType.APPLICATION_JSON;
+
+import org.jbehave.core.annotations.Given;
+import org.jbehave.core.annotations.Then;
+import org.jbehave.core.annotations.When;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Component;
+import org.symphonyoss.symphony.bots.helpdesk.service.makerchecker.client.MakercheckerClient;
+import org.symphonyoss.symphony.bots.helpdesk.service.model.Makerchecker;
+import org.symphonyoss.symphony.bots.helpdesk.service.model.UserInfo;
+
+/**
+ * Created by alexandre-silva-daitan on 23/02/18.
+ */
+
+@Component
+public class MakerCheckerSteps {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(MakerCheckerSteps.class);
+
+  @Autowired
+    private TestRestTemplate restTemplate;
+
+  @Given("a system state")
+  public void doSomething() {
+  }
+
+  @When("call the create makerchecker API")
+  public void callMakercheckerAPI() {
+
+    Makerchecker makerchecker = createMakerchecker();
+
+    ResponseEntity<Makerchecker> responseEntity = restTemplate.postForEntity("/v1/makerchecker" , makerchecker, Makerchecker.class);
+
+    assertEquals(HttpStatus.OK, responseEntity.getStatusCode());
```

Comment:
these assertions should be part of the 'then' clause.

Reply:
ok